migrations/900000000000058_add_default_syncs_for_container_images.up.sql

```diff
@@ -0,0 +1,205 @@
+('github-issues', 'Retrieves all the issues of a GitHub repo', 'docker', 'ghcr.io/mergestat/sync-github-issues', 'latest', 'github'),
+('github-pull-request-commits', 'Retrieves commits for all pull requests in a GitHub repo', 'docker', 'ghcr.io/mergestat/sync-github-pull-request-commits', 'latest', 'github'),
+('github-pull-request-reviews', 'Retrieves the reviews of all pull requests in a GitHub repo', 'docker', 'ghcr.io/mergestat/sync-github-pull-request-reviews', 'latest', 'github'),
+('github-pull-requests', 'Retrieves all the pull requests of a GitHub repo', 'docker', 'ghcr.io/mergestat/sync-github-pull-requests', 'latest', 'github'),
+('github-repo-info', 'Retrieves info/metadata about a GitHub repo', 'docker', 'ghcr.io/mergestat/sync-github-repo-info', 'latest', 'github'),
+('github-repo-stargazers', 'Retrieves all stargazers of a GitHub repo', 'docker', 'ghcr.io/mergestat/sync-github-repo-stargazers', 'latest', 'github'),
+('scan-gitleaks', 'Executes a gitleaks scan on a git repository', 'docker', 'ghcr.io/mergestat/sync-scan-gitleaks', 'latest', 'default'),
+('scan-gosec', 'Executes a gosec scan on a git repository', 'docker', 'ghcr.io/mergestat/sync-scan-gosec', 'latest', 'default'),
+('scan-grype', 'Executes a grype scan on a git repository', 'docker', 'ghcr.io/mergestat/sync-scan-grype', 'latest', 'default'),
+('scan-syft', 'Executes a syft scan on a git repository to generate an SBOM', 'docker', 'ghcr.io/mergestat/sync-scan-syft', 'latest', 'default'),
+('scan-trivy', 'Executes a trivy scan on a git repository', 'docker', 'ghcr.io/mergestat/sync-scan-trivy', 'latest', 'default'),
+('scan-yelp-detect-secrets', 'Executes a Yelp detect-secrets scan on a git repository', 'docker', 'ghcr.io/mergestat/sync-scan-yelp-detect-secrets', 'latest', 'default')
+ON CONFLICT DO NOTHING;
+
+--Adding more columns to the response of the function
+--https://www.graphile.org/postgraphile/computed-columns/
+CREATE OR REPLACE FUNCTION mergestat.container_syncs_latest_sync_runs(container_syncs mergestat.CONTAINER_SYNCS)
+RETURNS JSONB
+LANGUAGE PLPGSQL STABLE
+AS $$
+DECLARE
+   response JSONB;
+BEGIN
+    WITH last_completed_syncs AS(
+        SELECT
+            cs.id AS container_sync_id,
+            ci.name AS container_image_name,
+            j.id AS job_id,
+            j.status,
+            j.created_at,
+            j.started_at,
+            j.completed_at,
+            (SELECT COUNT(1) FROM sqlq.job_logs WHERE sqlq.job_logs.job = j.id AND level = 'warn') warning_count,
+            (SELECT COUNT(1) FROM sqlq.job_logs WHERE sqlq.job_logs.job = j.id AND level = 'error') error_count
+        FROM mergestat.container_syncs cs
+        INNER JOIN mergestat.container_sync_executions cse ON cs.id = cse.sync_id
+        INNER JOIN mergestat.container_images ci ON cs.image_id = ci.id
+        INNER JOIN sqlq.jobs j ON cse.job_id = j.id
+        WHERE cs.id = container_syncs.id
+        ORDER BY cs.id, j.completed_at DESC
```

Comment:
```suggestion
        ORDER BY cs.id, j.created_at DESC
```
@amenowanna it is important order by `created_at` here because it takes first 15 records, so we need to sort by `created_at` here as well to get it working

Reply:
Good call!